bin/functions

```diff
@@ -37,19 +38,19 @@ function check-jq() {
 }
 
 function retrieve-adoptopenjdk() {
-    URLS=("https://api.adoptopenjdk.net/v2/info/releases/{openjdk8}?type=jdk"
-          "https://api.adoptopenjdk.net/v2/info/releases/{openjdk9}?type=jdk"
-          "https://api.adoptopenjdk.net/v2/info/releases/{openjdk10}?type=jdk"
-          "https://api.adoptopenjdk.net/v2/info/releases/{openjdk11}?type=jdk"
-          "https://api.adoptopenjdk.net/v2/info/releases/{openjdk12}?type=jdk"
-          "https://api.adoptopenjdk.net/v2/info/releases/{openjdk13}?type=jdk")
+    URLS=("https://api.adoptopenjdk.net/v3/assets/feature_releases/{8}/ga?architecture=${ARCHITECTURE}&os=${OS}&image_type=jdk&page=0&page_size=100&project=jdk&sort_order=ASC&vendor=adoptopenjdk"
+          "https://api.adoptopenjdk.net/v3/assets/feature_releases/{9}/ga?architecture=${ARCHITECTURE}&os=${OS}&image_type=jdk&page=0&page_size=100&project=jdk&sort_order=ASC&vendor=adoptopenjdk"
+          "https://api.adoptopenjdk.net/v3/assets/feature_releases/{10}/ga?architecture=${ARCHITECTURE}&os=${OS}&image_type=jdk&page=0&page_size=100&project=jdk&sort_order=ASC&vendor=adoptopenjdk"
+          "https://api.adoptopenjdk.net/v3/assets/feature_releases/{11}/ga?architecture=${ARCHITECTURE}&os=${OS}&image_type=jdk&page=0&page_size=100&project=jdk&sort_order=ASC&vendor=adoptopenjdk"
+          "https://api.adoptopenjdk.net/v3/assets/feature_releases/{12}/ga?architecture=${ARCHITECTURE}&os=${OS}&image_type=jdk&page=0&page_size=100&project=jdk&sort_order=ASC&vendor=adoptopenjdk"
+          "https://api.adoptopenjdk.net/v3/assets/feature_releases/{13}/ga?architecture=${ARCHITECTURE}&os=${OS}&image_type=jdk&page=0&page_size=100&project=jdk&sort_order=ASC&vendor=adoptopenjdk")
 
     # shellcheck disable=SC2046
     if [[ -z "$(ls -A "${CACHE_DIR}"/adopt-*.json 2>/dev/null)" ]] || [[ $(set -- $(${STAT}) && echo "${1}") -le $(( $(date +%s) - 3600)) ]]
     then
         for url in "${URLS[@]}"
         do
-            curl -skL "${url}" -# -w "%{filename_effective}\n" -o "${CACHE_DIR}/adopt-#1.json" > /dev/null 2>&1
```

Comment:
I've reverted #55 because the script should fail if the HTTPS certificates are invalid, which `-k` (`--insecure`) prevents.